source/pick-from-possibly-undefined.d.ts

```diff
@@ -0,0 +1,23 @@
+/**
+It allows to `Pick` properties from a type that may be undefined
+
```

Comment:
need add Use-case

> Write about some real-world use-cases where it can be useful. (It can be hard sometimes for users to see where they would use something)
https://github.com/sindresorhus/type-fest/blob/main/.github/contributing.md

Reply:
Now, better?